src/features/content-answers/ContentAnswersModalFooter.tsx

```diff
@@ -34,7 +42,7 @@ const ContentAnswersModalFooter = ({ intl }: Props) => {
     return (
         <div className="ContentAnswersModalFooter">
             <div className="ContentAnswersModalFooter-avatar">
-                <Avatar />
+                <Avatar id={id} name={name} />
```

Comment:
I see that eua uses the first initial. Could we change this to `name={name.charAt(0)}` instead.

Reply:
I have to change the way Avatar functions in order to get a single initial to come out, functionally when its one word, it will just use that initial twice in a row based on the test

Reply:
https://github.com/box/box-ui-elements/blob/master/src/components/avatar/__tests__/AvatarInitials.test.tsx#L14

Reply:
https://github.com/box/box-ui-elements/blob/665d30a26683ad7a2a0668e23e23434fb22a3f8d/src/components/avatar/__tests__/AvatarInitials.test.tsx#L14